content/departments/product-engineering/product/process/pricing.md

```diff
@@ -57,26 +57,32 @@ This section is to be determined; in the meantime please work with Product Marke
 This section is to be determined; in the meantime please work with Product Marketing and Product Leadership to determine the price for new features that will be charged as add-ons.
 
 ## How pricing and packaging decisions are made
-Pricing and packaging recommendations should be driven by those that are closest to the work. 
-- For new products and features, this is typically Product Management and Product Marketing. 
+
+Pricing and packaging recommendations should be driven by those that are closest to the work.
+
+- For new products and features, this is typically Product Management and Product Marketing.
 - For changes to how we charge or how much we charge, this may be Sales, CE, Sales Ops, or BizOps.
 
-At times, it may make sense for a working group to spin up to address our pricing and packaging (example: Cloud for teams). 
+At times, it may make sense for a working group to spin up to address our pricing and packaging (example: Cloud for teams).
 
 Given the potential impact of pricing and packaging changes across all teams, it is important that a cross-functional group is able to review all proposed changes. While other people and teams should certainly be consulted for feedback/approval, be sure to include the following people:
+
 - Marketing: Kacie, Andy
 - Sales: Gregg
 - Product: Christina
 - Ops: Dan, Steph
 
-While the entire council should be consulted, there will typically be only one or two final approvers. The pricing council can help determine who the approver should be (and that person may not be part of the pricing council). 
+While the entire council should be consulted, there will typically be only one or two final approvers. The pricing council can help determine who the approver should be (and that person may not be part of the pricing council).
```

Comment:
How do you make a request for a decider to the pricing council? Email the group? Create a ticket? Ask in Slack? Something else?

Reply:
+1! The way we did this our time around was meet 1:1 with people in every category here (Gregg, Christina, Steph, Andy) and ultimately land at which person (Gregg in our case) needed to approve the pricing. 

Perhaps a quick process for not only where to ask, but also what you should have ready when you make the ask would be helpful? From my experience, it was helpful to have the conversations about who should approve the final decision after doing a lot of the initial work / initial RFC, so council members could get a sense of which areas were the biggest question areas (for example: are the main questions about product and where to draw the free / not free line? Are the main questions about the dollar price? Are the main questions about packaging?)